### PR TITLE
Follow the change of ONNX Cast operator "to" attribute

### PR DESCRIPTION
**Pull request by bddppq**
This needs to go into master together with https://github.com/onnx/onnx/pull/727

### CODE REVIEW

**Review by houseroad — APPROVED** [review]
LGTM, thanks.

**Comment by bddppq** [comment]
The failures in CI are expected since it needs the pairing change in onnx/onnx#727.

@ezyang could you take a look at the changes in torch/onnx/symbolics.py?

**Review by yinghai — APPROVED** [review]
Looks good.

File: torch/onnx/symbolic.py [hunk 1]
```diff
@@ -631,19 +631,19 @@ def _unique(g, input, sorted, return_inverse):
...
 cast_pytorch_to_onnx = {
-    'uint8_t': 'UINT8',
-    'int8_t': 'INT8',
-    'double': 'DOUBLE',
-    'float': 'FLOAT',
-    'Half': 'FLOAT16',
-    'int': 'INT32',
-    'int64_t': 'INT64',
-    'int16_t': 'INT16',
+    'uint8_t': 2,
```
Comment (ezyang) [inline torch/onnx/symbolic.py]: This makes me sad. Can we get these integers from the protobuf bindings?

**Review by ezyang — APPROVED** [review]

**Pull request merged**